### ISSUE TITLE
Is Mouse GO slim Developed by MGI available?

### ISSUE THREAD

**Issue by vitkl** [geneontology/helpdesk#167]
Link to "Mouse GO slim Developed by MGI" points to http://www.geneontology.org/page/go-subset-guide rather than *.obo file. Could you please explain if this is a bug or whether this SLIM ontology is currently unavailable?

**Comment by kltm** [comment]
@vitkl 
Our apologies for the trouble. Looking at the parent directory (http://www.geneontology.org/ontology/subsets/), there seems to be no mouse slim available. @ukemi , would you happen to know the point person for the generation or maintenance of the mouse slim?

**Comment by cmungall** [comment]
http://purl.obolibrary.org/obo/go/subsets/goslim_mouse.obo

Web page needs updated

**Comment by pgaudet** [comment]
> Web page needs updated

Right, but since we're moving the website, I only updated it on the new pages. I  was expecting the switch to be happening soon; if you thin it won't, I will update the Drupal site as well. 

Pascale

**Comment by kltm** [comment]
@pgaudet There is no reason to have that information out of date and the exact release date of the new site is unclear--I'd go ahead and update the current page as well.

**Comment by ukemi** [comment]
@mdolanme is the contact person for the mouse slim.
